scripts.js

```diff
@@ -38,36 +38,33 @@ function imageSetup (img) {
   const outputDiv = document.getElementById('output-display');
   outputDiv.appendChild(outputCanvas);
   
-  listenerStuff(curiosity);
+  SetupUpdateEvents(curiosity);
 };
 
+function SetupUpdateEvents(curiosity) {
+  const updateAfterSliderChange = (event) => {
+    const sliderOutputToBeUpdated = document.getElementById(`${event.target.id}-output`);
+    sliderOutputToBeUpdated.value = event.target.value;
 
-
-
-// mainScript();
-
-function listenerStuff(curiosity) {
-  const update = (event) => {
+    const curiosityKeyToUpdate = event.target.dataset.key;
```

Comment:
in the future it may make sence to only declare new variables at the top of a function, but this is only a code lintting preference not important for functionality for the moment

Reply:
Why does it make sense?
I disagree from a readability perspective.

Reply:
multiline code snipets!!! thank you @khutten for showing thats posible.
apparently you can do syntax highlighting if you wanted to try it is below.
```javascript
function iAmAFunction(){
  const iAmAThing
}
```

I agree with that your blocks of code read very well.
however I think i like at the beginning of a context understanding all the variables at play is helpful. If you want to have variables declared mid flow than you may as well contain them within a new context. This can be done via puting inside a function. However likely you'd prefer to just wrap within a context. I haven't done it in a codebase myself but i remember learning it when i first learnt ES6. from memory also it will not contain var's only es6 declarations like const and let.

Example
```javascript
// main context
const imAThang = document.getElementById('hello');

{
  // newContext where we can declare new variables to make alistairs where style he made up happy
  const imAnotherThang = 'I am only accessible within this context';

  // wont contain var's i think (could be wrong here)
  var imAScaryThang = 'I think i will still be accessible outside of this context'

  
}
```

please note all this is from memory when i first learnt it 2 years ago, and i haven't tested any of it.